src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -1,4 +1,5 @@
 (function(global, doc, $, React, ReactDOM, eZ, Routing) {
+    const SELECTOR_BULK_OPERATION_FAILED_MODAL = '.ez-bulk-operation-failed-modal';
     const listContainers = [...doc.querySelectorAll('.ez-sil')];
```

Comment:
Can you convert it to: `const listContainers = doc.querySelectorAll('.ez-sil');`?